tests/test_help.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8
+
+import sys
+
+import pytest
+
+from requests.help import info
+
+
+@pytest.mark.skipif(sys.version_info[:2] != (2,6), reason="Only run on Python 2.6")
+def test_system_ssl_py26():
+    """OPENSSL_VERSION_NUMBER isn't provided in Python 2.6, verify we don't
+    blow up in this case.
+    """
+    assert info()['system_ssl'] == {'version': ''}
```

Comment:
Yep, that seems reasonable if we're going to be testing for this specifically. I've added that into the test commit, if we decide to merge that part of the PR.